src/sentry/eventstore/base.py

```diff
@@ -6,76 +6,107 @@
-    DEVICE_NAME = Column("device_name", None, "device.name")
-    DEVICE_BRAND = Column("device_brand", None, "device.brand")
-    DEVICE_LOCALE = Column("device_locale", None, "device.locale")
-    DEVICE_UUID = Column("device_uuid", None, "device.uuid")
-    DEVICE_ARCH = Column("device_arch", None, "device.arch")
-    DEVICE_BATTERY_LEVEL = Column("device_battery_level", None, "device.battery_level")
-    DEVICE_ORIENTATION = Column("device_orientation", None, "device.orientation")
-    DEVICE_SIMULATOR = Column("device_simulator", None, "device.simulator")
-    DEVICE_ONLINE = Column("device_online", None, "device.online")
-    DEVICE_CHARGING = Column("device_charging", None, "device.charging")
-    GEO_COUNTRY_CODE = Column("geo_country_code", None, "geo.country_code")
-    GEO_REGION = Column("geo_region", None, "geo.region")
-    GEO_CITY = Column("geo_city", None, "geo.city")
-    ERROR_TYPE = Column("exception_stacks.type", None, "error.type")
-    ERROR_VALUE = Column("exception_stacks.value", None, "error.value")
-    ERROR_MECHANISM = Column("exception_stacks.mechanism_type", None, "error.mechanism")
-    ERROR_HANDLED = Column("exception_stacks.mechanism_handled", None, "error.handled")
-    STACK_ABS_PATH = Column("exception_frames.abs_path", None, "stack.abs_path")
-    STACK_FILENAME = Column("exception_frames.filename", None, "stack.filename")
-    STACK_PACKAGE = Column("exception_frames.package", None, "stack.package")
-    STACK_MODULE = Column("exception_frames.module", None, "stack.module")
-    STACK_FUNCTION = Column("exception_frames.function", None, "stack.function")
-    STACK_IN_APP = Column("exception_frames.in_app", None, "stack.in_app")
-    STACK_COLNO = Column("exception_frames.colno", None, "stack.colno")
-    STACK_LINENO = Column("exception_frames.lineno", None, "stack.lineno")
-    STACK_STACK_LEVEL = Column("exception_frames.stack_level", None, "stack.stack_level")
-    CONTEXTS_KEY = Column("contexts.key", "contexts.key", "contexts.key")
-    CONTEXTS_VALUE = Column("contexts.value", "contexts.value", "contexts.value")
+    EVENT_ID = Column("event_id", "event_id", "event_id", "id")
+    GROUP_ID = Column("group_id", None, "group_id", "issue.id")
+    ISSUE = Column("issue", None, "group_id", "issue.id")
+    PROJECT_ID = Column("project_id", "project_id", "project_id", "project.id")
+    TIMESTAMP = Column("timestamp", "finish_ts", "timestamp", "timestamp")
+    TIME = Column("time", "bucketed_end", "time", "time")
+    CULPRIT = Column("culprit", None, "culprit", "culprit")
+    LOCATION = Column("location", None, "location", "location")
+    MESSAGE = Column("message", "transaction_name", "message", "message")
+    PLATFORM = Column("platform", "platform", "platform", "platform.name")
+    ENVIRONMENT = Column("environment", "environment", "environment", "environment")
+    RELEASE = Column("tags[sentry:release]", "release", "sentry:release", "release")
```

Comment:
Should this name be `release` for discover?

Reply:
Yeah, updated now